api/src/services/eml-service.ts

```diff
@@ -464,6 +465,27 @@ export class EmlService extends DBService {
     };
   }
 
+  /**
+   * Builds the EML Dataset section for a survey
+   *
+   * @param {string} packageId
+   * @param {SurveyObject} surveyData
+   * @return {*}  {Record<string, any>}
+   * @memberof EmlService
+   */
+  _buildSurveyEmlDatasetSection(packageId: string, surveyData: SurveyObject): Record<string, any> {
+    return {
+      $: { system: EMPTY_STRING, id: packageId },
+      title: surveyData.survey_details.survey_name,
+      creator: this._getSurveyContact(surveyData),
```

Comment:
nit: I'm hoping this will just lead to more clarity for future maintainers: Could you create a `_getSurveyCreator` function that gets called here, and then that function can just call the getSurveyContact function and return its result. Mainly just to have distinct functions for all the sections, to hopefully assist with understanding the EML.

We sort of did the same thing for the `_getSurveyAdditionalMetadata` where it exists and gets called, even though it returns nothing. Just to maintain the placeholder for it.